lib/inertia_rails/base_prop.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module InertiaRails
+  # Base class for all props.
+  class BaseProp
```

Comment:
I like this. Half-baked thought: I wonder if we could move more rendering logic into the `Prop` classes. The `computed_props` method is doing more and more things in a sort of procedural style. Maybe it could iterate over the props one time and each instance would know what to do with itself.

(Not suggesting changes, more just thinking out loud)